lstchain/datachecks/containers.py

```diff
@@ -326,35 +327,44 @@ def fill_pixel_wise_info(self, table, mask, histogram_binnings,
-        # Account for average relative spot shift (outwards) due to coma
-        # aberration:
-        relative_shift = 1.0466 # For LST's paraboloid
-        camera_frame = CameraFrame(telescope_pointing=pointing,
-                                   focal_length=focal_length*relative_shift,
-                                   obstime=obstime,
-                                   location=LST1_LOCATION)
-        telescope_frame = TelescopeFrame(obstime=obstime, location=LST1_LOCATION)
-
-        # radius around star within which we consider the pixel may be affected
-        # (hence we will later not raise a flag if e.g. its pedestal std dev is
-        # high):
-        r_around_star = 0.25 * u.deg
-        stars = bright_stars['ra_dec']
-        pixels = SkyCoord(x=geom.pix_x, y=geom.pix_y,
-                          frame=camera_frame).transform_to(telescope_frame)
-        angular_distance = pixels[:, np.newaxis].separation(stars)
-
-        # This counts how many stars are close to each pixel; stars can be
-        # counted more than once (for different pixels!) so don't add them up.
-        self.num_nearby_stars = np.count_nonzero(angular_distance < r_around_star,
-                                                 axis=1)
+
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore",
+                                    category=NonRotationTransformationWarning)
+            warnings.filterwarnings("ignore", category=ErfaWarning)
+
+            horizon_frame = AltAz(location=LST1_LOCATION, obstime=obstime)
+            pointing = SkyCoord(az=self.mean_az_tel,
+                                alt=self.mean_alt_tel,
+                                frame=horizon_frame)
+            bright_stars = get_bright_stars(time=obstime,
+                                            pointing=pointing, radius=3*u.deg,
+                                            magnitude_cut=8)
+            # Account for average relative spot shift (outwards) due to coma
+            # aberration:
+            relative_shift = 1.0466 # For LST's paraboloid
+            camera_frame = CameraFrame(telescope_pointing=pointing,
+                                       focal_length=focal_length*relative_shift,
+                                       obstime=obstime,
+                                       location=LST1_LOCATION)
+            telescope_frame = TelescopeFrame(obstime=obstime, location=LST1_LOCATION)
+
+            # radius around star within which we consider the pixel may be affected
+            # (hence we will later not raise a flag if e.g. its pedestal std dev is
+            # high):
+            r_around_star = 0.25 * u.deg
+            stars = bright_stars['ra_dec']
+
+            pixels = SkyCoord(x=geom.pix_x, y=geom.pix_y,
+                              frame=camera_frame).transform_to(telescope_frame)
+
+            angular_distance = pixels[:, np.newaxis].separation(stars)
```

Comment:
Ah, the warning here comes from the `separation` call. Instead of silencing the warning, I think you can avoid it entirely be doing `.separation(stars.transform_to(telescope_frame))`

Reply:
get_bright_stars also raises the warnings. Opened issue in ctapipe: https://github.com/cta-observatory/ctapipe/issues/2800